packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -248,15 +248,15 @@ export const DetailsTile = ({
   }
 
   const imageElement = coSign ? (
-    <CoSign size={Size.LARGE} style={styles.coverArt}>
+    <CoSign size={Size.LARGE}>
       <DynamicImage
-        source={{ uri: imageUrl }}
+        source={imageUrl ? { uri: imageUrl } : undefined}
```

Comment:
nice check here, so would we need to do this for all dynamic images that can take a uri that may be undefined?

Reply:
Yeah, otherwise it doesn't load the fallback gradient. I thought about baking it into DynamicImage but ImageSourceType is fairly wide. It could still be done though

Reply:
yeah totally agree its hard cause of the type ^^